exercises/nth-prime/example.jl

```diff
@@ -0,0 +1,43 @@
+function isPrime(x)

```

Comment:
This should be `isprime` to follow Julia's code style conversion and the style guide outlined in the README of this repo.